app/gui2/src/components/GraphEditor/GraphNodes.vue

```diff
@@ -32,6 +37,13 @@ function moveNode(movedId: ExprId, delta: Vec2) {
 function hoverNode(id: ExprId | undefined) {
   if (selection != null) selection.hoveredNode = id
 }
+
+const uploadingFiles = computed<[FileName, File][]>(() => {
+  const currentStackItem = projectStore.executionContext.getStackTop()
+  return [...projectStore.awareness.allUploads()].filter(
+    (value) => value[1].stackItem == currentStackItem,
```

Comment:
What is under `[1]` index in value? Maybe we could make params like `([, someMeaningfulName]) => someMeaningfulName.stackItem ...`?